docs/pages/operations-guide.md

```diff
@@ -252,6 +252,38 @@ Configuring `ROOT` vs `MOUNT` volumes may depend on the service. Some services w
 
 A Task generally maps to a process. A Pod is a collection of Tasks that share an environment. All Tasks in a Pod will come up and go down together. Therefore, [restart](#restart-a-pod) and [replace](#replace-a-pod) operations are at Pod granularity rather than Task granularity.
 
+## Overlay networks
+
+The SDK supports `pods` joining the `dcos` overlay network. You can specify that a pod should join the overlay by adding the following to your service spec YAML:
+
+```yaml
+pods:
+  pod-on-overlay:
+    count: {{COUNT}}
+    # join the 'dcos' overlay network
+    networks:
+      dcos:
+    tasks:
+      ...
+  pod-on-host:
+    count: {{COUNT}}
+    tasks:
+      ...
+```
+
+This has a few direct effects:
+  * Every pod gets its own IP address and its own array of ports
+  * Pods do not use the ports on the host machine
+  * Pod IP addresses can be resolved with the DNS: `<task_name>.<framework_name>.autoip.dcos.thisdcos.directory`
+
+There are a few indirect effects that a framework architect should know:
```

Comment:
remove that entirely? "Specifying that pod join the `dcos` overlay network has the following indirect effects:"